### PR TITLE
eos_ospfv3 resource module

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY
Closes https://github.com/ansible-collections/arista.eos/issues/69

##### ISSUE TYPE

- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: tests/unit/modules/network/eos/fixtures/eos_ospfv3_config_gather.cfg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+router ospfv3
```
Comment (NilashishC) [inline tests/unit/modules/network/eos/fixtures/eos_ospfv3_config_gather.cfg:1]: Should this be indented?

File: changelogs/fragments/109-eos-ospfv3-resource-module.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
```
Comment (NilashishC) [inline changelogs/fragments/109-eos-ospfv3-resource-module.yaml:2]: Should this be a minor change? I'm not sure if `major_changes` would entail a major version bump.

**Comment by NilashishC** [comment]
Not a blocker but coverage for rm_templates/ospfv3 is at 40%, it'd be nice to increase that in the future.

**Comment by GomathiselviS** [comment]
> Not a blocker but coverage for rm_templates/ospfv3 is at 40%, it'd be nice to increase that in the future.



> Not a blocker but coverage for rm_templates/ospfv3 is at 40%, it'd be nice to increase that in the future.

ya , I will work on it.

**Review by rohitthakur2590 — APPROVED** [review]

**Pull request merged**